CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Add `--strict` option to mod add and descriptor commands, STCLI-59
 * Correct get-stdin dependency, fixes STCLI-60
 * Use Node 8 to be in line with stripes-core requirement
+* Highlight failed git-pull attempts in a dumb-terminal-friendly way.
```

Comment:
Version 1.2 was just released last week.  This can be first on the list for 1.3!